doc/book/admin/upgrades/2.11.0.rst

```diff
@@ -0,0 +1,27 @@
+..  _admin-upgrades-2.8.1_mixed_transactions:
+
+Recover from WALs with mixed transactions when upgrading to 2.11.0
+==================================================================
+
+This is a guide on fixing a specific problem that could happen when upgrading
+from a Tarantool version between 2.1.2 and 2.2.0 to 2.8.1 or later. The described
+solution is applicable since version 2.11.0.
+
+The problem is described in the issue :tarantool-issue:`7932`. If two or more
+transactions happened simultaneously in Tarantool 2.1.2-2.2.0, their operations
+could be written to the write-ahead log mixed with each other. Starting from version
+2.8.1, Tarantool recovers transactions atomically and expects all WAL entries
+between a transaction's ``begin`` and ``commit`` operations to belong to one transaction.
+If there is an operation belonging to another transaction, Tarantool fails to recover
+from such a WAL.
+
+Starting from version :doc:`2.11.0 </release/2.11.0>`, Tarantool can recover from
+WALs with mixed transactions in the ``force_recovery`` mode.
+
+To restore data from a WAL with mixed transactions in a replica set:
+
+1.  Stop read-only instances and delete their WALs.
```

Comment:
Let's rewrite this guide a bit. There are two separate problems:

1st problem: all or part of instances updated to 2.11 won't start due to recovery error with mixed transactions
In this case all the user has to do is start the faulty nodes with `force_recovery` = true.
After recovering, the user has to make a snapshot on all the faulty nodes, so that the broken xlogs are not used for recovery anymore. After that `force_recovery` may be reset to false.

2nd problem:
May or may not emerge after the 1st one is solved. Here some of the replicas can't replicate from other replicas, because they are sending broken xlogs.
In this case indeed the user has to rebootstrap all the replicas, who can't connect to others (because others have new data, but in a broken xlog).
After the rebootstraps are done, everything should work as expected.